core-bundle/src/Image/PictureFactory.php

```diff
@@ -192,10 +192,10 @@ private function createConfigItem($imageSize): PictureConfigurationItem
 
         if (null !== $imageSize) {
             $resizeConfig
-                ->setWidth($imageSize->width)
-                ->setHeight($imageSize->height)
+                ->setWidth((int) $imageSize->width)
+                ->setHeight((int) $imageSize->height)
                 ->setMode($imageSize->resizeMode)
```

Comment:
Should we string-cast this as well, just to be sure (e.g. not get `null`)?

Reply:
An empty string would also result in an exception, so the cast is not necessary.